test/test_train_cifar.py

```diff
@@ -19,6 +19,12 @@
 import unittest
 
 
+writer = None
```

Comment:
In test_util.py:

def get_gummary_write(logdir):
  if logdir:
    ...
    return ...

Reply:
Done.